docs/content/Upgrade/metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Monitoring Upgrade Guidance
+date:
+draft: false
+weight: 100
+---
+
+# Upgrade Guidance for PostgreSQL Operator Monitoring
+
+## Migration to Upstream Containers
+
+The Crunchy PostgreSQL Monitoring Stack now uses upstream Prometheus and Grafana
+containers. By default the installers will deploy the monitoring stack using
+images from Dockerhub but can easily be updated to point to a Redhat cerified
+container repository. The Redhat certified image catalog can be found
+[here](https://catalog.redhat.com/software/containers/explore) and the Dockerhub
+images can be found at the following links:
+
+- https://hub.docker.com/r/prom/prometheus
+- https://hub.docker.com/r/grafana/grafana
+- https://hub.docker.com/r/prom/alertmanager
+
+These containers are configurable through Kubernetes ConfigMaps and the updated
+PostgreSQL Operator Monitoring installers. Once deployed Prometheus and Grafana
+will be populated with resource data from metrics-enabled PostgreSQL clusters.
+
+## New Monitoring Features
+
+### Alerting
+The updated PostgreSQL Operator Monitoring stack supports deployment of 
+Prometheus Alertmanager. This deployment uses upstream Prometheus
+Alertmanager images that can be installed and configured with the metrics
+installers and Kubernetes ConfigMaps.
+
+### Updated pgMonitor
+Prometheus and Grafana have been updated to include a default configuration from
+[pgMonitor](https://github.com/CrunchyData/pgmonitor) that is tailored for
+container-based PostgreSQL deployments. This updated configuration will show
+container specific resource information from your metrics-enabled PostgreSQL
+clusters. By default the metrics stack will include:
+
+- New Grafana dashboards tailored for container-based PostgreSQL deployments
+- Container specific operating system metrics
+- General PostgreSQL alerting rules.
+
+## Updated Monitoring Installer
+The installer for the PostgreSQL Operating Monitoring has been split out into
+a separate set of installers. With each installer (Ansible, a Kubectl job,
+or Helm) you will be able to apply custom configurations through Kubernetes
+ConfigMaps.
```

Comment:
Thinking we can specify a quick list of things that can be customized, e.g.:

> With each installer (Ansible, a Kubectl job, or Helm) you will be able to apply custom configurations through Kubernetes ConfigMaps.  This includes:
> 
> - Custom Grafana dashboards
> - Custom Prometheus scrape configuration
> - Custom alerting rules

Reply:
what do you think of this @andrewlecuyer?

- Custom Grafana dashboards and datasources
- Custom Prometheus scrape configuration
- Custom Prometheus alerting rules
- Custom Alertmanager notification configuration

Reply:
Sounds good to me!